cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_ListSecurityGroupsRequest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v2.securitygroups;
+
+import org.cloudfoundry.Nullable;
+import org.cloudfoundry.client.v2.InFilterParameter;
+import org.cloudfoundry.client.v2.IsFilterParameter;
+import org.cloudfoundry.client.v2.PaginatedRequest;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+/**
+ * The request payload for the List all Security Groups operation
+ */
+@Value.Immutable
+abstract class _ListSecurityGroupsRequest extends PaginatedRequest {
+
+    /**
+     * The names
+     */
+    @InFilterParameter("name")
+    @Nullable
+    abstract List<String> getNames();
+
+    /**
+     * The running default
+     */
+    @IsFilterParameter("running_default")
```

Comment:
Hi @twoseat - You are right. The [doc](https://apidocs.cloudfoundry.org/latest-release/security_groups/list_all_security_groups.html) only says that 'name' is a valid filter.
I will remove invalid filters. Sorry about that.